8/debian-10/rootfs/opt/bitnami/scripts/libdrupal.sh

```diff
@@ -280,6 +289,27 @@ drupal_site_install() {
     fi
 }
 
+drupal_create_config_directory() {
+	if [[ -z $DRUPAL_CONFIG_SYNC_DIR ]]; then
+		# In order to use Drush commands we need to have a value for
+		# config_sync_directory or it will throw an error so we add a
+		# temp value here first
+		drupal_conf_set "\$settings['config_sync_directory']" "temp" no
+		DRUPAL_CONFIG_SYNC_DIR="sites/default/files/config_$(drush eval "echo(Drupal\Component\Utility\Crypt::randomBytesBase64(55))")"
+		debug_execute mkdir "$DRUPAL_BASE_DIR/$DRUPAL_CONFIG_SYNC_DIR"
+	else
+		debug_execute mkdir "$@"
+	fi
```

Comment:
We have some common functions for improving the readability of scripts, such as `is_empty_value`, `generate_random_string` or `ensure_dir_exists`. To improve the readability of this function you could rewrite it as:

```suggestion
    local config_sync_dir="${DRUPAL_CONFIG_SYNC_DIR:-}"
    if is_empty_value "$config_sync_dir"; then
        config_sync_dir="${DRUPAL_BASE_DIR}/sites/default/files/config_$(generate_random_string -t alphanumeric -c 16)"
    fi
    ensure_dir_exists "$config_sync_dir"
```

That way, there is no dependency with Drush and you don't need to set a false value.